website_product_share/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Product Share
+=============================
+
+Allow user to enable/disable share button on each product page
```

Comment:
It's a global option. I assume that if user want a product by product then he/she could add a share button snippet. This module will be useful if website has a significant number of product. Adding share button on each product page will be painfull